### PR TITLE
Fixing kotlin duplicate class error

### PR DESCRIPTION
**Pull request by deannagarcia**
Removing exports of common proto files from Kotlin to fix duplicate class error #8694

### CODE REVIEW

**Review by fowles — APPROVED** [review]

**Review by Iftimasum1 — APPROVED** [review]
Iftimasum motalib

**Comment by JStr84galaxy** [comment]
> Removing exports of common proto files from Kotlin to fix duplicate class error #8694

File: java/kotlin-lite/pom.xml [hunk 1]
```diff
@@ -68,24 +68,6 @@
   </dependencies>
 
   <build>
-     <!-- Include core protos in the bundle as resources -->
```
Comment (JStr84galaxy) [inline java/kotlin-lite/pom.xml:71]: https://github.com/protocolbuffers/protobuf/releases/download/v3.17.3/protobuf-all-3.17.3.tar.gz

**Review by JStr84galaxy — APPROVED** [review]

**Review by JStr84galaxy — APPROVED** [review]

**Review by JStr84galaxy — APPROVED** [review]

**Review by JStr84galaxy — APPROVED** [review]

**Review by JStr84galaxy — APPROVED** [review]
java/kotlin-lite/pom.xml

**Review by JStr84galaxy — APPROVED** [review]

**Review by JStr84galaxy — COMMENTED** [review]
java/kotlin/pom.xml

**Review by JStr84galaxy — COMMENTED** [review]
java/kotlin/pom.xml

**Review by JStr84galaxy — COMMENTED** [review]
deannagarcia:morePomFixes

**Review by JStr84galaxy — COMMENTED** [review]
deannagarcia:morePomFixes

**Review by JStr84galaxy — COMMENTED** [review]
deannagarcia:morePomFixes

**Review by JStr84galaxy — COMMENTED** [review]
deannagarcia:morePomFixes

**Review by JStr84galaxy — COMMENTED** [review]
deannagarcia:morePomFixes

**Pull request merged**